src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -154,8 +159,19 @@ static boolean isNew(TenantAttributes attributes, String featureVersion) {
     return future.compose(result -> runJavaMigrations(attributes, vertxContext, headers).map(result));
   }
 
+  @Validate
+  public void postTenant(TenantAttributes tenantAttributes, Map<String, String> headers,
+                         Handler<AsyncResult<Response>> handler, Context context) {
+    // delete Kafka topics if tenant purged
+    Future.succeededFuture()
+      .compose(x -> tenantAttributes.getPurge()
```

Comment:
A tenant can be disabled with purge=true or without purge=true: https://github.com/folio-org/okapi/blob/master/doc/guide.md#disabling

For me the intention of https://issues.folio.org/browse/MODINVSTOR-827 is to delete the kafka topics in both cases. Disabling without purge=true allows to upgrade to a new version without the need for keeping the kafka topics in the old version.

Reply:
@julianladisch  Can you please share an example of such requests for disabling without prge=true

Reply:
@julianladisch we decided to delete topics only on purge